css-three-column-layout/styles.css

```diff
@@ -0,0 +1,74 @@
+.container{
+  max-width: 1000px;
+  margin-left: auto;
+  margin-right: auto;
+}
+
+.row{
+  display: flex;
+}
+
+.blue-background{
+  background-color: #1896d9;
+}
+
+.column-one-third{
+  flex-basis: calc(100%/3);
+}
+
+.column-two-thirds{
+  flex-basis: calc(100%/3);
+}
+
+.column-third{
+  flex-basis: calc(100%/3);
+}
```

Comment:
Why do you need a `column-one-third` and ` column-third` class? Also, the measurement for `column-two-thirds` is not accurate. These details **_really_** matter.